src/components/form/select/select.test.tsx

```diff
@@ -63,4 +63,20 @@ describe('<Select />', () => {
 
     expect(value).toBeInstanceOf(HTMLElement)
   })
+
+  it('should contains input with required attribute', () => {
```

Comment:
You have tested when the value is `true`, now, you can test if the value is `false`.